CHANGELOG.md

```diff
@@ -5,6 +5,71 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Implement JcsTezosSignature2021.
+- Add PrimaryDIDURL for dereference trait method.
+- Add EthereumEip712Signature2021 v1 context.
+- Add VerificationMethodMap::get_id.
+- Implement converting JWK to Tezos format.
+- Add did-pkh-poly.
+- Pass resolver option in issue/prepare.
+- Use vc-test-suite example keys and DIDs.
+- Update blockchainAccountId for new CAIP-10.
+- Add Revocation List 2020 context.
+- Validate key and algorithm for JsonWebSignature2020.
+- Work-in-progress implementation of detached JWS decoding.
+- Default to PS256 for RSA.
+- Implement PS256.
+- Work-in-progress ZCap invocation methods
+- Implement FromStr for URI.
+- Support publicKeyJwk for EthereumEip712Signature.
+- Add DID Test Suite implementation generator.
+- Implement EthereumPersonalSignature2021.
+- Implement EthereumEip712Signature2021.
+- Disallow untyped properties in eip712.
+- Enable Eip712Signature2021 in did:pkh.
+- Allow using Eip712 proof with normal VM types.
+- Add did-webkey and SSH key parsing.
+- Implement JWK Thumbprint.
+- Use compressed P-256 did:key.
+- Add UserAgent header in requests.
+- Add ES256K to alg mistmatch handling.
+- Use custom-made JWK algorithms for Tezos.
+- Canonicalize negative zero.
```

Comment:
This one ("Canonicalize negative zero.") was a bug fix rather than a feature, so should probably be listed under Fixed.